editor/components/block-list/index.js

```diff
@@ -250,7 +250,7 @@ class BlockList extends Component {
 
 		return (
 			<BlockSelectionClearer>
-				{ !! blocks.length && <BlockListSiblingInserter /> }
+				{ !! blocks.length && <BlockInsertionPoint /> }
```

Comment:
When will this insertion point ever be shown?

Reply:
It is supposed to be shown when you open the inserter in an empty post but I just tried and it's not showing up. Might be an issue somewhere else.

Reply:
Oh! I guess it's the inversed condition and also some styling issues.

On a second thought, might be it's not needed when the post is empty.